Create gui-not-working.md

jdorel opened the pull request.

andrewdavidwong commented:
We might consider adding this to https://www.qubes-os.org/doc/gui-configuration/ rather than creating a new, separate page.

jdorel commented:
> We might consider adding this to https://www.qubes-os.org/doc/gui-configuration/ rather than creating a new, separate page.

I only saw the title `GUI Configuration` in the page list. Just moved it to the correct page, also renamed it.
